src/main/java/seedu/bankwithus/Account.java

```diff
@@ -0,0 +1,28 @@
+package seedu.bankwithus;
+
+public class Account {
+    protected String name;
+    protected float balance;
+
+
+    /**
+     * @param name initialise in the name of the account
```

Comment:
Would appreciate if a description is given. Even a single like of trivial description will do